Resources/config/assetic.xml

```diff
@@ -25,6 +25,9 @@
     </parameters>
 
     <services>
+        
+        <defaults public="true" />
```

Comment:
`<defaults public="true" />` is not compatible with SF2.3. But set each service can be set as public.